ui/component/hiddenNsfwClaims/view.jsx

```diff
@@ -9,14 +10,24 @@ type Props = {
 };
 
 export default (props: Props) => {
-  const { numberOfHiddenClaims, obscureNsfw } = props;
+  const { numberOfHiddenClaims: number, obscureNsfw } = props;
 
+  function getMessage() {
+    if (number > 1) return __('%number% files', { number });
+    return __('%number% file', { number });
+  }
   return (
     obscureNsfw &&
-    Boolean(numberOfHiddenClaims) && (
+    Boolean(number) && (
       <div className="section--padded section__subtitle">
-        {numberOfHiddenClaims} {numberOfHiddenClaims > 1 ? __('files') : __('file')} {__('hidden due to your')}{' '}
-        <Button button="link" navigate="/$/settings" label={__('content viewing preferences')} />.
+        <I18nMessage
+          tokens={{
+            settings_link: <Button button="link" navigate="/$/settings" label={__('content viewing preferences')} />,
+            message: getMessage(),
+          }}
+        >
+          %message% hidden due to your %settings_link%.
```

Comment:
{ number > 1 ? '%number% files hidden due to your %settings_link%' : %number% file hidden due to your %settings_link%' }

Reply:
```{ number > 1 ? '%number% files hidden due to your %content_viewing_preferences_link%' : %number% file hidden due to your %content_viewing_preferences_link%' }```